### ISSUE TITLE
inputbox don't send the text 

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#346]
*Issue migrated from trac ticket [#62](http://colloquy.info/project/ticket/62)*

**type:** Defect | **component:** Colloquy (Mac) | **severity:** Normal | **priority:** Normal | **version:** Local Build | **status:** closed | **resolution:** Fixed
___
#### 2005-03-14 19:40:48: knuterik@… created the issue

had a problem with the text on the input box not getting sent when i hit return, but only on the connection to freenode, where i could not send to any channel or to console. the inputboxes for my simultanious connection to quakenet worked just fine.  i have had this error happening several. Disconnecting freenode and reconnecting using the "connections" window, seemed to fix it after a couple of attemts, while quitting colloquy and restarting it gave me the same error on the same connection (freenode). I'm have set colloquy to connect automatically to quakenet and the freenode at startup.
___
#### 2005-03-17 23:39:36: novaa commented

I also see this sometimes (also with a simultaneous connection to Quakenet where everything works), mostly a restart of colloquy helps.  I can say that its not Terminal.app with its secure input (which bugged me with bf-1942)  If I find a way to reproduce this i will post it here.

___
#### 2005-04-04 18:25:04: @akempgen commented

i also have this problem from time to time. iirc it happens only after waking from sleep here, a manual reconnect or autoconnect on startup dont seem to cause it...

___
#### 2005-05-13 03:23:03: @xeenon changed status from *new* to *closed*

___
#### 2005-05-13 03:23:03: @xeenon changed resolution from ** to *fixed*

___
#### 2005-05-13 03:23:03: @xeenon commented

I think this has been fixed for a while. Reopen it is happens again.

___
#### 2006-03-08 18:44:57: @autoxr commented

Happens to me with latest nightly and experimental. Doesn't happen often, but it'll randomly happen when I connect to my proxy.  Maybe 1 in 10 or 20 times. Sometimes disconnecting/reconnecting works, sometimes it doesn't and then I have to quit relaunch. That usually clears it up.

___
#### 2006-03-08 18:44:57: @autoxr changed status from *closed* to *reopened*

___
#### 2006-03-08 18:44:57: @autoxr changed version from *2.0 (2D9)* to *Built Source*

___
#### 2006-03-08 18:44:57: @autoxr changed resolution from *fixed* to **

___
#### 2006-06-14 05:17:52: @xeenon commented

Are there any exceptions in the system console when this problem occurs?

___
#### 2006-07-10 18:07:15: @akempgen commented

making #366 a duplicate of this one

___
#### 2006-07-15 02:55:40: @akempgen commented

jruderman had
```
2006-07-14 13:40:56.949 Colloquy10205 setGroupIdentifier:: _NSUndoStack 0x4f1260 is in invalid state, calling setGroupIdentifier with no begin group mark
```
in his console during this bug

___
#### 2007-09-25 05:04:58: @xeenon commented

Also see #1105

___
#### 2007-09-25 08:36:00: @xeenon changed status from *reopened* to *closed*

___
#### 2007-09-25 08:36:00: @xeenon changed resolution from ** to *fixed*

___
#### 2007-09-25 08:36:00: @xeenon commented

Fixed in 92aff5d1ab1e1bbc6f97e134c7107a274786d077.
